packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js

```diff
@@ -174,16 +173,11 @@ function CollapsiblePanelHeader(props) {
 
 function getKeyValueContent(content) {
 	return (
-		<dl className={css.content}>
-			{content.map((item, index) => [
-				<dt className={css.label} key={`${index}_label`}>
-					<Label>{item.label}</Label>
-				</dt>,
-				<dd className={css.description} key={`${index}_desc`}>
-					{item.description}
-				</dd>,
+		<div className={css.content}>
```

Comment:
I was in favor of Definition List here

Reply:
I'm asking ux teams

Reply:
you're right. we should use label instead of badges. waiting for new labels

Reply:
✅